web-ng/src/app/components/layer-dialog/layer-dialog.component.html

```diff
@@ -14,7 +14,12 @@
   limitations under the License.
 -->
 
-<h2 mat-dialog-title>Customize layer</h2>
+<inline-editor 
+  mat-dialog-title
+  class="layer-name"
+  [data]="layerName" 
+  placeholder='Unnamed layer' 
+  (focusOut)='saveLayerName($event)'></inline-editor>
```

Comment:
Naming: Perhaps `setLayerName` would be more appropriate, since it doesn't actually write to the db?